include/appimage/libappimage.h

```diff
@@ -0,0 +1,31 @@
+#include <glib.h>
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Return the md5 hash constructed according to
+* https://specifications.freedesktop.org/thumbnail-spec/thumbnail-spec-latest.html#THUMBSAVE
+* This can be used to identify files that are related to a given AppImage at a given location */
+char *get_md5(char const *path);
```

Comment:
I think that we still need the "extern" on every declaration, I'll test it latter

Reply:
It wasn't necessary before obviously, when you used it from C++, but you might be right, at least for the C interface (for C++, the `extern` has been defined already).